docs/content/deploy.md

```diff
@@ -28,98 +24,58 @@ To start a local deployment, either:
 
 ## Cloud deployment
 
+To start a cloud deployment follow the section.
+
+### Setup the SpatialOS project name
+
 You are allocated an empty SpatialOS project in the cloud when you sign up to SpatialOS; you use this to deploy your game, but to do this you need to tell the GDK the name of your allocated SpatialOS project so it knows where to deploy your game to.
 
-To start a cloud deployment:
-1.  Open the `spatialos.json` file in the root folder of your game.
+1. Open the `spatialos.json` file in the root folder of your game.
 1. Change the `name` field so it matches the name of your SpatialOS project. You can find this in the SpatialOS [Console](https://console.improbable.io). It’ll be something like `beta_someword_anotherword_000`.
-1. Build and upload a game assembly for the deployment
-    - The assembly includes executable files for the client-workers and server-workers, and the assets both types of workers use (such as the models and textures used by the client-server - that is, the game executable code - to visualise the game).
-    1. To build an assembly; in the Unity Editor, select **SpatialOS** > **Build all workers for cloud**.
 
-    1. To upload an assembly; open a terminal and navigate to the directory in which your game is in (the repository you’ve cloned). Run `spatial cloud upload <assembly name>`.
-    - The `<assembly name>` is a label you create so you can identify this assembly in the next step - for example you could call it `MyGDKAssembly`.
-    > **It’s finished uploading when:** You see a successful upload report printed in your terminal output, and your browser will open a console.improbable.io page dispaying the deployment.
+### Build and upload the game assembly
+The assembly includes executable files for the client-workers and server-workers, as well as the assets both types of workers use. (The assets might be the models and textures that the client-servers - that is, the game executable code - use to visualise the game.)
+
+  1. To build an assembly; in the Unity Editor, select **SpatialOS** > **Build all workers for cloud**.
+  1. To upload an assembly; open a terminal and navigate to the directory your game is in (the repository you’ve cloned). Run `spatial cloud upload <assembly name>`. The `<assembly name>` is a label you create so you can identify this assembly in the next step - for example you could call it `MyGDKAssembly`.
+
+  > **It’s finished uploading when:** You see a successful upload report printed in your terminal output, and your browser will open a console.improbable.io page dispaying the deployment.  Your browser automatically opens [https://console.improbable.io](https://console.improbable.io) displaying the deployment.
```

Comment:
we could cut this down to

"..., and your browser automatically opens a [https://console.improbable.io](https://console.improbable.io) page displaying the deployment."